src/icon/icon.jsx

```diff
@@ -40,14 +46,15 @@ const Icon = ({
   )
 
   // Label component
-  const label = labelFactory(iconCode)
+  const label = labelFactory(iconCode, alt)
 
   // Final component
   return (
     <div style={style} className={classes}>
-      {labelLeft && label(labelLeft)}
+      {!alt && labelLeft && label(labelLeft)}
```

Comment:
NO !!!! we want the label even if `alt` is provided, but should be hidden for TA in this case